docs/scos/user/back-office-user-guides/202200.0/merchandising/discount/creating-cart-rules.md

```diff
@@ -15,48 +15,241 @@ related:
+
+      {% info_block warningBox "Note" %}
 
-* On the **Discount calculation** tab, do the following:
-    1.  In the **CALCULATOR TYPE** drop-down, select either **Percentage** or **Fixed amount**. For reference information, in the **Create new discount** reference, see the [Discount calculation tab](/docs/scos/user/back-office-user-guides/{{page.version}}/merchandising/discount/creating-vouchers.html#discount-calculation-tab) section.
+      The next step varies based on the calculator type that you select:
+      * **Fixed amount**: Enter the amounts (per currency and price mode, if applicable) to be discounted.
+      * **Percentage**: Enter the percent value to be discounted.
 
-        {% info_block warningBox "Note" %}
+      {% endinfo_block %}
 
-        The next step varies based on the selected calculator type:
+   2. Select the **DISCOUNT APPLICATION TYPE** and define which products are eligible for the discount.
+   3. To proceed to the **Conditions** tab, click **Next**.
+4. On the **Conditions** tab, do the following:
+    1. Select the **APPLY WHEN** conditions or click **Plain query** and enter the query manually.
+    2. Enter the value for **THE DISCOUNT CAN BE APPLIED IF THE QUERY APPLIES FOR AT LEAST X ITEM(S).** field.
+5. To create the new discount, click **Save**.
 
-          a. *Fixed amount*: Enter the amounts (per currency and price mode, if applicable) to be discounted.
+### Reference information: Creating cart rule discounts
 
-          b. *Percentage*: Enter the percent value to be discounted.
+This section contains reference information you select and enter when creating cart rule discounts.
 
-        {% endinfo_block %}
+#### General information tab
 
-    2. Select the **DISCOUNT APPLICATION TYPE** and define which products are eligible for the discount. For reference information, in the **Create new discount** reference, see the [Discount calculation tab](/docs/scos/user/back-office-user-guides/{{page.version}}/merchandising/discount/creating-vouchers.html#discount-calculation-tab) section.
- * Click **Next**, or select the **Conditions** tab to proceed.
- * On the **Conditions** tab, do the following:
-    1. Select the **APPLY WHEN** conditions or click **Plain query** and enter the query manually. For reference information, see [Conditions](/docs/scos/user/back-office-user-guides/{{page.version}}/merchandising/discount/creating-vouchers.html#conditions).
-    2. Enter the value for **THE DISCOUNT CAN BE APPLIED IF THE QUERY APPLIES FOR AT LEAST X ITEM(S).** field.
-* To create the new discount, click **Save**.
+The following table describes the attributes you enter and select on the **General information** tab:
+
+<div class="width-100">
+
+| ATTRIBUTE |DESCRIPTION  |
+| --- | --- |
+| STORE RELATION | Stores that your cart rule will be active in. You can select multiple stores.|
+| DISCOUNT TYPE | Drop-down list where you select either **Voucher codes** or **Cart rule** discount type. |
```

Comment:
Explain what these discount types are. Give links to the respective sections in the feature overview.

Reply:
Done